backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/graphql/GraphqlDatabaseFieldFactory.java

```diff
@@ -6,14 +6,14 @@
 
 import graphql.Scalars;
 import graphql.schema.*;
+import java.net.URL;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
-import org.molgenis.emx2.Constants;
-import org.molgenis.emx2.Database;
-import org.molgenis.emx2.SchemaInfo;
+import org.molgenis.emx2.*;
```

Comment:
suggest explicit imports

Reply:
this is done automatically by our stylechecker